lib/utils/acars_free_text.test.ts

```
@@ -0,0 +1,13 @@
+
+describe('acars free text helper', () => {
+  test('cleans whitespace', () => {
+    expect(formatAcarsFreeText('  HELLO   WORLD  ')).toBe('HELLO WORLD');
+  });
+
+  test('handles empty input', () => {
+    expect(formatAcarsFreeText('')).toBe('');
+    expect(formatAcarsFreeText(null)).toBe('');
+    expect(formatAcarsFreeText(undefined)).toBe('');
+  });
+});
```

ESLint

[error] Insert `⏎`

(prettier/prettier)